telemetry/packages/public-app/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@hyped/public-app",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev -p 3001",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@preact/signals-react": "^1.3.6",
+    "@radix-ui/react-switch": "^1.0.3",
+    "@tremor/react": "^3.9.2",
+    "class-variance-authority": "^0.7.0",
+    "clsx": "^2.0.0",
+    "date-fns": "^2.30.0",
+    "i": "^0.3.7",
```

Comment:
what is this package for?

Reply:
Something to do with styling using className - not too sure

https://www.linkedin.com/pulse/creating-react-components-style-variants-using-class-variance-aliyu-ixymf/